command/login_test.go

```diff
@@ -82,7 +87,7 @@ func TestLoginCommand_Run(t *testing.T) {
 			t.Fatal(err)
 		}
 
-		if l, exp := len(storedToken), vault.TokenLength+2; l != exp {
+		if l, exp := len(storedToken), minTokenLengthExternal+vault.TokenPrefixLength; l != exp {
```

Comment:
Ah, well my thinking was that the only thing that contributes to variable lengths is the compression of the uint_32 fields and the potential resulting variable signature. But here because the test server is up for such a small amount of time and so few writes happen, none of this ever really changes on oss or ent, so it seemed ok to use an exact length check. We could probably change it to >= to be more precise and make the test more robust, but I haven't seen any flakiness here so far.

Reply:
Please add a comment in the test then.